worker/include/RTC/TransportCongestionControlServer.hpp

```diff
@@ -89,6 +98,12 @@ namespace RTC
 		uint8_t unlimitedRembCounter{ 0u };
 		std::deque<double> packetLossHistory;
 		double packetLoss{ 0 };
+		SeqNumUnwrapper<uint16_t> unwrapper;
+		absl::optional<int64_t> periodicWindowStartSeq;
+		// Map unwrapped seq -> time.
+		std::map<int64_t, uint64_t> packetArrivalTimes;
+		// Use buffer policy similar to webrtc
```

Comment:
Dot at the end of comments.
```suggestion
		// Use buffer policy similar to webrtc.
```